docs/UserGuide.md

```diff
@@ -183,6 +183,16 @@ Examples:
 * `list` followed by `memo 2 m/meet at 6pm` adds the memo “meet at 6pm” to the 2nd person in the address book. 
 * `find Betsy` followed by `memo 1 m/meet at 6pm` adds the memo “meet at 6pm” to the 1st person in the results of the `find` command.
 
+### Invoke recent command :`invoke`
+
+Invoke the most recent command being used.
```

Comment:
Just nitpicking here but to remain consistent, shouldn't this say "Invokes" instead of "Invoke"?